Vagrantfile

```diff
@@ -149,8 +149,6 @@ Vagrant.configure("2") do |config|
         vb.cpus = vm_cpus i
         vb.customize ["modifyvm", :id, "--cpuexecutioncap", "#{$vb_cpuexecutioncap}"]
         config.ignition.config_obj = vb
-
-        vb.customize ['modifyvm', :id, '--nictype2', '82545EM']
```

Comment:
what was this needed for before? are we losing functionality now with this removal?

Reply:
I put that change in to switch the NIC type so that the virtio_net module wouldn't be used, because the virtuio_net module was randomly trying to do a kernel paging request on the RX path, which would cause an immediate reboot.  I didn't change both NIC types because on my linux/virtualbox environment that caused the order of the NICs to change, causing problems for bring up networking in coreos.  It appears that having this change causes NIC reordering in the mac/virtualbox environment.  Ugh.